plugin/build.gradle

```diff
@@ -104,8 +106,9 @@ compileTestJava {
 dependencies {
     api group: 'org.springframework', name: 'spring-beans', version: "${spring_version}"
     api "com.fasterxml.jackson.core:jackson-core:${jackson_version}"
-    api "com.fasterxml.jackson.core:jackson-databind:${jackson_version}"
-    api "com.fasterxml.jackson.core:jackson-annotations:${jackson_version}"
+    // Bundled with the OpenSearch (https://github.com/opensearch-project/OpenSearch/pull/5366)
+    compileOnly "com.fasterxml.jackson.core:jackson-databind:${jackson_version}"
```

Comment:
Changing the dependency scope because of https://github.com/opensearch-project/OpenSearch/commit/c6663fd7f4e74b3ed1b6ad70e758b3e6bd21f816